docs/testing.rst

```diff
@@ -81,3 +81,76 @@ When the reason for the breackage is identified there are two options to go forw
     * If the failure comes from an intended change in behaviour adjust Anaconda or the tests
     * If the failure uncovers an actual regression, file bugs for the given Fedora components. If it does not look like the issue will be
       fixed quickly, work around in Anaconda or add a `naughty override file <https://github.com/cockpit-project/bots/tree/main/naughty/>`_, thus marking the expected failure pattern.
+
+Fix Web UI pixel tests
+----------------------
+
+So pixel tests fail on your PR - now what ?
+
+There are essentially two options:
+
+    * the test uncovered a in your PR bug that needs to be fixed in the code
+    * your PR changes the visual behavior of the Web UI in a valid manner and the pixel test needs to be fixed
+
+We will cover the second option.
+
+First make sure your PR is rebased on latest revision of the target branch. This is important as the issue you are seeing might have already been fixed by another PR that has since been merged.
+
+If rebasing your PR did not fix the issue, and pixel tests still fail in CI on your PR, you will need to update one or more of the reference images.
+
+::
+
+    $ cd ui/webui
+    $ GITHUB_BASE=rhinstaller/anaconda ./test/common/pixel-tests pull
```

Comment:
I am getting `bash: ./test/common/pixel-tests: No such file or directory`. Have you tried it with a clean repository?

Reply:
Do `make test/common` between `cd` and invoking the script.

Reply:
I don't have a Makefile by default:
`make: Nothing to be done for 'test/reference'.`

This seems to work:
`make -f Makefile.am test/reference`

Reply:
Ouch, I didn't realize that.

This feels more like investigative journalism than documenting...

Reply:
Yep, that works on a clean checkout - steps updated.

Still, this should not be needed. IMHO, we really should scrap these nested makefiles or at least move the most often needed targets to the top-level makefile. Also ideally, all the targets should run all necessary initialization steps if at all possible, when invoked.

Otherwise we are going to drown in "and where do I run this ?" and "what do I need to run before running this ?" issues.

Reply:
I don't really care about nested makefiles as long as there is a good documentation.